litgpt/eval/evaluate.py

```diff
@@ -69,6 +69,21 @@ def convert_and_evaluate(
 
     from lm_eval import evaluator
 
+    if tasks is None:
+        from lm_eval.tasks import TaskManager
+        taskm = TaskManager()
+
+        for key, value in taskm.task_index.items():
+            print(key)
+
+        print(
+            "To evaluate multiple tasks, you can chain the task names "
+            "listed above via a comma-separated list,"
+            "\nfor example: `--tasks 'hellaswag,truthfulqa_mc2,mmlu'`. "
```

Comment:
Feels odd to continue with lowercase after a breakline

```suggestion
            "listed above via a comma-separated list."
            "\nFor example: `--tasks 'hellaswag,truthfulqa_mc2,mmlu'`. "
```